services/cadip/rs_server_cadip/cadip_retriever.py

```diff
@@ -1,22 +1,50 @@
 """This will be used to configure dataretriever used for cadip"""
 import json
+import os
 import os.path as osp
+import tempfile
 from pathlib import Path
 from typing import Any
 
-from services.common.rs_server_common.data_retrieval.data_retriever import DataRetriever
-from services.common.rs_server_common.data_retrieval.download_monitor import (
-    DownloadMonitor,
-)
-from services.common.rs_server_common.data_retrieval.eodag_provider import EodagProvider
-from services.common.rs_server_common.data_retrieval.provider import (
-    CreateProviderFailed,
-)
-from services.common.rs_server_common.data_retrieval.storage import Storage
+from rs_server_common.data_retrieval.data_retriever import DataRetriever
+from rs_server_common.data_retrieval.download_monitor import DownloadMonitor
+from rs_server_common.data_retrieval.eodag_provider import EodagProvider
+from rs_server_common.data_retrieval.provider import CreateProviderFailed
+from rs_server_common.data_retrieval.storage import Storage
+from rs_server_common.utils.logging import Logging
 
 CONF_FOLDER = Path(osp.realpath(osp.dirname(__file__))).parent / "config"
 EODAG_CONFIG = Path(osp.realpath(osp.dirname(__file__))).parent / "config" / "cadip_ws_config.yaml"
 
+logger = Logging.default(__name__)
+
+
+def get_eodag_provider(station) -> str:
```

Comment:
In services/cadip/config/cadip_ws_config.yaml (see above) replace template values by environment variables